static/js/components/EditFollowupRequestDialog.jsx

```diff
@@ -40,6 +40,27 @@ const EditFollowupRequestDialog = ({
     handleClose();
   };
 
+  // Set default form values to current request data
+  const { formSchema } = instrumentFormParams[
+    followupRequest.allocation.instrument.id
+  ];
+
+  Object.keys(formSchema.properties).forEach((key) => {
+    formSchema.properties[key].default = followupRequest.payload[key];
+  });
+
+  const validate = (formData, errors) => {
+    if (
+      formData.start_date &&
+      formData.end_date &&
+      formData.start_date > formData.end_date
```

Comment:
Just double checking whether this is a valid way to compare these objects.

Reply:
You're right, probably better to make these into Date objects and compare. It should work as is with the string comparison since the dates submitted via the frontend are stored in `YYYY-MM-DD` format, but I guess people could always submit weird stuff via the API.